### PR TITLE
ALL Tables added

### PR DESCRIPTION
**Pull request by sinha-arpita**

### CODE REVIEW

File: data/migrations/20190515102553_FriendJoinTable.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('friends', function(friends){
+        friends.increments();
+        friends.string('friend_name',128).notNullable();
```
Comment (alex-bezruchko) [inline data/migrations/20190515102553_FriendJoinTable.js]: We only need reference keys from other tables (no need for strings). Here we only need user_id and friend_id (both of them from the users table).

File: data/migrations/20190515111950_LinkTable.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
+exports.up = function(knex, Promise) {
+return knex.schema.createTable('links', function(links){
+links.increments();
+links.integer('event_id');//declaring a foreign key
+links.foreign('event_id')//using the foreign key
```
Comment (alex-bezruchko) [inline data/migrations/20190515111950_LinkTable.js]: Line 6 is not necessary. The integer event_id is referencing the id from the events table.

File: data/migrations/20190515111950_LinkTable.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    .references('id')
+    .inTable('events');
+
+
+links.integer('user_id');//declaring the other foreign key
+links.foreign('user_id')//using this foreign key
```
Comment (alex-bezruchko) [inline data/migrations/20190515111950_LinkTable.js]: Line 6 is not necessary. The integer user_id is referencing the id from the users table.

File: data/migrations/20190515115207_LocationTable.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable("locations", function(locations) {
+        locations.increments();
+        locations
+          .string("location_name").notNullable();
```
Comment (alex-bezruchko) [inline data/migrations/20190515115207_LocationTable.js]: We won't need that. The google api already gives that info.

File: data/migrations/20190515165804_EventTable.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('events',function(events){
+      events.increments();
+      events.string("event_name");
+      events.string("event_description");
+      events.string("event_date").notNullable();
+      //foreign keys
+      events.integer("organiser").notNullable();
+      
+      events.foreign("organiser").references("id").inTable("users");
```
Comment (alex-bezruchko) [inline data/migrations/20190515165804_EventTable.js]: remove foreign property. replace with user_id

File: data/migrations/20190515165804_EventTable.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+      events.integer("place").notNullable();
+      events.foreign("place").references("id").inTable("locations");
```
Comment (alex-bezruchko) [inline data/migrations/20190515165804_EventTable.js]: remove foreign property. replace with place_id.

File: data/migrations/20190515172744_CommentTable.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable("comments",function(comments){
+           comments.increments();
+           comments.string("time").notNullable();
+           comments.text("message");
+           //foreign keys
+           comments.integer("event_id").notNullable();
+           comments.foreign("event_id").references("id").inTable("events");
+            
+           comments.integer("event_date").notNullable();
```
Comment (alex-bezruchko) [inline data/migrations/20190515172744_CommentTable.js]: we won't need line 10 & 11. we will join the tables in the db Helper

File: data/migrations/20190515172744_CommentTable.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+           comments.foreign("event_date").references("event_date").inTable("events");
+            
+           comments.integer("user_id").notNullable();
```
Comment (alex-bezruchko) [inline data/migrations/20190515172744_CommentTable.js]: no need for foreign prop.

**Review by alex-bezruchko — DISMISSED** [review]
Please see the comments and update the tables.

File: data/migrations/20190515102553_FriendJoinTable.js (hunk 1 above)
Comment (TaylorBelk1) [reply data/migrations/20190515102553_FriendJoinTable.js]: here we don't need friends name

**Review by BoxOfCereal — APPROVED** [review]
I think things look good! Lets merge this!

**Review by BoxOfCereal — APPROVED** [review]

**Pull request merged**